website/content/docs/ui/custom-login.mdx

```diff
@@ -0,0 +1,546 @@
+---
+layout: docs
+page_title: Manage custom login settings
+description: >-
+  Use custom login flows to control how clients can authenticate to Vault
+  through the GUI.
+---
+
+# Manage custom login settings for the Vault GUI
+
+@include 'alerts/enterprise-and-hcp.mdx'
+
+Login settings can be used to customize which methods display in the web UI login form by setting a default and back up login methods. 
+Available to be created via the CLI or HTTP API.
+
+## Before you start 
+
+@include 'alerts/restricted-admin.mdx'
+
+- **You must have Vault Enterprise 1.20.0 or higher installed.**
+- **You must have the appropriate permissions**:
+  - To **create login settings**, you must have `create` permission for the [`sys/config/ui/login/default-auth`](/vault/api-docs/system/config-ui-login-default-auth) endpoint.
+  - To **edit login settings**, you must have `update` permission for the [`sys/config/ui/login/default-auth`](/vault/api-docs/system/config-ui-login-default-auth) endpoint.
+  - To **delete login settings**, you must have `delete` permission for the [`sys/config/ui/login/default-auth`](/vault/api-docs/system/config-ui-login-default-auth) endpoint.
+
+## Step 1: Select your authentication method
+
+Only the following methods are supported by the GUI and can be set as a default or backup method:
+
+@include 'ui/supported-login-methods.mdx'
+
+You can customize your login flow with a single default authentication method, multiple
+backup authentication methods, or both.
+
+Configuration           | Result
+----------------------- | ------
+Default only            | Users see only the selected authentication method during login.
+Backup(s) only          | Users see only from the selected authentication method(s) during login.
+Default and backup(s)   | Users see the default method first but can access the backup methods as alternate options.
+
+
+## Step 2: Make the authentication method visible to the GUI. (Optional, but recommended) 
+
+Set the [listing visibility](/vault/api-docs/system/auth#listing_visibility-1) of your authentication plugin to `unauth` 
+so the Vault GUI knows which login URL to use for authentication.
+
+<Warning title="Listing visibility and plugin mount paths">
+
+Login customizations specify authentication **types**, not specific plugin paths.
+As a result, the Vault GUI assumes each login method is mounted at its default [path](/vault/api-docs/system/auth#path), 
+which corresponds to its auth type.
+
+If a login customization specifies a type for a plugin mounted at a **non-default** path, 
+that mount **must** be configured with `listing_visibility=unauth`.
+
+Otherwise, users will be unable to authenticate through the customized login view.
+
+</Warning>
```

Comment:
```suggestion
```
Doubling up on asides isn't a good idea. We can just make the visibility information part of the document text